tests/components/cover/test_command_line.py

```diff
@@ -44,41 +42,42 @@ def test_query_state_value(self):
 
     def test_state_value(self):
         """Test with state value."""
-        with tempfile.TemporaryDirectory() as tempdirname:
-            path = os.path.join(tempdirname, 'cover_status')
-            test_cover = {
-                'command_state': 'cat {}'.format(path),
-                'command_open': 'echo 1 > {}'.format(path),
-                'command_close': 'echo 1 > {}'.format(path),
-                'command_stop': 'echo 0 > {}'.format(path),
-                'value_template': '{{ value }}'
-            }
-            self.assertTrue(setup_component(self.hass, cover.DOMAIN, {
-                'cover': {
-                    'platform': 'command_line',
-                    'covers': {
-                        'test': test_cover
-                    }
+        test_cover = {
+            'command_state': 'cat cover_status',
+            'command_open': 'echo 1 > cover_status',
+            'command_close': 'echo 1 > cover_status',
+            'command_stop': 'echo 0 > cover_status',
+            'value_template': '{{ value }}'
+        }
+        self.assertTrue(setup_component(self.hass, cover.DOMAIN, {
```

Comment:
`setup_component` almost always returns `True` since #3736. Please use the following:

```python
from tests.common import assert_setup_component

-snip-

with assert_setup_component(1):
    assert setup_component(....)
```